lib/src/syn/v1/kind.rs

```diff
@@ -89,6 +90,7 @@ fn geometry(i: &str) -> IResult<&str, Kind> {
 fn array(i: &str) -> IResult<&str, Kind> {
 	let (i, _) = tag("array")(i)?;
 	let (i, v) = opt(|i| {
+		let (i, _) = mightbespace(i)?;
```

Comment:
This one is not essential to the fix right? I don't mind it tagging along but just to make sure I understand it correctly 😅

Reply:
Oh, huh, I thought I had removed that again. No it is not part of the fix but it is a bit strange that `record <foo>` is allowed but `array <foo>` is not.

Reply:
This brings array more inline with record.

Reply:
Yep! Absolutely! I don't mind it sneaking in but if you want to open a separate PR for it then that's fine by me too!